text/0000-build-std/6-appendix-summary-of-changes.md

```diff
@@ -0,0 +1,97 @@
+# Appendix I: Summary of changes
+[appendix-i]: #appendix-i-summary-of-changes
+
+There are many features proposed in this RFC for different parts of the project:
+
+- Bootstrap/infra/release
+  - [Vendoring standard library sources into `rust-src`](./4-build-std-always.md#vendored-rust-src) (`build-std=always`)
+  - [`rust-src` is a default component](./4-build-std-always.md#vendored-rust-src) (`build-std=always`)
+  - [`rust-self-contained` components](./4-build-std-always.md#self-contained-objects) (`build-std=always`)
+  - [Testing build-std in rust-lang/rust CI][constraints-on-the-standard-library]
+- Cargo
+  - [`build-std = "always"`](./4-build-std-always.md) (`build-std=always`)
+    - [Extending Cargo subcommmands](./4-build-std-always.md#cargo-subcommands)
+  - [Prohibiting custom targets](./4-build-std-always.md#custom-targets) (`build-std=always`)
+  - [Explicit dependencies](./5-standard-library-dependencies.md#proposal) (*Standard library dependencies*)
+    - [Lockfile changes](./5-standard-library-dependencies.md#proposal)
+    - [Registry changes](./5-standard-library-dependencies.md#registries)
+    - [Extending Cargo subcommmands](./5-standard-library-dependencies.md#cargo-subcommands)
+- Compiler
+  - [Loading `panic_unwind` from `-L dependency=`](./4-build-std-always.md#proposal) (`build-std=always`)
+  - [`--no-implicit-sysroot-deps`](./4-build-std-always.md#preventing-implicit-sysroot-dependencies) (`build-std=always`)
+  - [Destabilise custom targets](./4-build-std-always.md#custom-targets) (`build-std=always`)
+  - [Assuming `RUSTC_BOOTSTRAP` for sysroot builds](./4-build-std-always.md#building-the-standard-library-on-a-stable-toolchain) (`build-std=always`)
+  - [Detect missing `rust-self-contained` components and provide diagnostics](./4-build-std-always.md#self-contained-objects) (`build-std=always`)
+  - [Forcing many codegen-units for `compiler-builtins`](./4-build-std-always.md#compiler-builtins) (`build-std=always`)
+- Project-wide
+  - [Documenting build-std stability guarantees](./4-build-std-always.md#stability-guarantees) (`build-std=always`)
+- Standard library
+  - [Removing `restricted_std`](./4-build-std-always.md#restricted_std) (`build-std=always`)
+  - [Moving configuration into the standard library's profile](./4-build-std-always.md) (`build-std=always`)
+
+## Constraints on the standard library, compiler and bootstrap
+[constraints-on-the-standard-library]: #constraints-on-the-standard-library-compiler-and-bootstrap
+
+A stable mechanism for building the standard library imposes some constraints on
+the rest of the toolchain that would need to be upheld:
+
+- No further customisation of the pre-built standard library through any means
```

Comment:
```suggestion
- No further required customisation of the pre-built standard library through any means
```

I think we probably are OK with *optional* customization, right?

Reply:
I think so, changed in 7bfca0b1